docs/topics/whatsnew-eap.md

```diff
@@ -104,14 +104,83 @@ We'd appreciate your feedback in any form:
 
 ## New Kotlin/Wasm target
 
-> The new backend is available in EAP of IntelliJ IDEA 2022.3 only.
->
-{type="note"}
+Kotlin/Wasm (Kotlin WebAssembly) goes [Experimental](components-stability.md#stability-levels-explained) in this preview release!
+Kotlin team finds [WebAssembly](https://webassembly.org/) is promising technology. We have a close look at it and 
+try to find better ways how you can use WebAssembly and get all the benefits of Kotlin.
+
+WebAssembly binary format is independent of the platform because it runs using its own virtual machine.
+Almost all the modern browsers support WebAssembly in an experimental mode, so you don't need to install additional tooling to 
+have an environment to run WebAssembly, therefore, Kotlin/Wasm.
+
+There are three main advantages of the new Kotlin/Wasm that we want to highlight:
 
-Kotlin/Wasm goes [Alpha](components-stability.md) in this preview release.
+* Performance: fast speed of the compilation process
+* Faster integration with the host
+* Small size of the final binary
+
+Since 1.8.20-Beta preview release, you can use Kotlin/Wasm in your experimental projects.
+We provide Kotlin standard library (`stdlib`) and test library (`kotlin.test`) for Kotlin/Wasm out of the box.
 
 [Learn more about Kotlin/Wasm in the YouTube video](https://www.youtube.com/watch?v=-pqz9sKXatw).
 
+### How to enable the Kotlin/Wasm
+
+To create a new Kotlin/Wasm project with Project Wizard, install the EAP version of IntelliJ IDEA 2022.3 and select the
+**Browser application with Kotlin/Wasm** project in the **Kotlin Multiplatform** section.
+
+To enable the Kotlin/Wasm and test it without IDE support, update your `build.gradle.kts` file:
+
+```kotlin
+plugins {
+   kotlin("multiplatform") version "1.8.20-Beta"
+}
+
+kotlin {
+   wasm {
+       binaries.executable()
+       browser {
+           commonWebpackConfig {
+           }
+       }
+   }
+   sourceSets {
+       val commonMain by getting
+       val commonTest by getting {
+           dependencies {
+               implementation(kotlin("test"))
+           }
+       }
+       val wasmMain by getting
+       val wasmTest by getting
+   }
+}
+```
+
+> Check out the [GitHub repository with Kotlin/Wasm examples](https://github.com/Kotlin/kotlin-wasm-examples).
+> 
+{type="note"}
+
+To run the Kotlin/Wasm project, you need to update the settings of the target environment:
+
+* **Chrome**. For version 109 or newer:
+   
+   1. Go to `chrome://flags/#enable-webassembly-garbage-collection` in your browser.
+   2. Relaunch the browser application.
+
+* **Firefox**. For version 111 or newer:
+
+   1. Go to `about:config` in your browser.
+   2. Enable `javascript.options.wasm_function_references` and `javascript.options.wasm_gc` options.
+   3. Relaunch the browser application.
+
+* **Edge**.
```

Comment:
For version 109 or newer:

Reply:
Other option: is merging this with Chrome section, thought it still doesn't have flag in chrome://flags/